src/components/Header.js

```diff
@@ -0,0 +1,15 @@
+import { Link } from 'react-router-dom';
+import '../styles/Header.css';
+
+const Header = () => (
+  <div className="header">
+    <h1>Math Magicians</h1>
+    <div className="link-container">
+      <Link to="/" className="link">Home</Link>
+      <Link to="/calculator" className="link">Calculator</Link>
+      <Link to="/quote" className="link">Quote</Link>
```

Comment:
- [ ] [OPTIONAL] React router provides a special type of `Link` called `NavLink` which makes it easier to apply styles on the currently active link. Please check the [documentation](https://reactrouter.com/en/main/components/nav-link) for more information. 😎️